objax/util/util.py

```diff
@@ -164,3 +165,13 @@ def to_tuple(v: Union[Tuple[Number, ...], Number, Iterable], n: int):
         return (v,) * n
     else:
         return tuple(v)
+
+
+def to_upsample(upsample: Union[UpSample, str]) \
```

Comment:
Can you put an `assert` at the end to inform the user they didn't pass an Upsample or str argument?

Reply:
sure @david-berthelot, i will look into them.